packages/shared/src/components/Settings.tsx

```diff
@@ -117,6 +131,13 @@ export default function Settings({
           checked={insaneMode}
           className="mx-1.5"
           onToggle={toggleInsaneMode}
+          disabled={!isLaptop}
+          tooltip={
+            !isLaptop && {
+              content: 'Layout will be fixed for mobile and tablet',
```

Comment:
Did you check this copy with Sab, sounds a bit funky.
Would expect: Layout is fixed for mobile and tablet view

Reply:
Just adjusted this copy from the other copy given, let me confirm.
Checking here: https://dailydotdev.slack.com/archives/C02AA18RD1D/p1717655445750759

Reply:
ok so will totally remove this for mobile and tablet as confirmed with Sab